lir/registry.py

```diff
@@ -19,9 +19,12 @@ def _get_attribute_by_name(name: str) -> Any:
     """Resolve the corresponding function or class in this project from the configuration string."""
     parts = name.split('.')
 
+    logging.debug(f'resolving class name from key: {name}')
```

Comment:
Zoals het er nu staat, wordt alles naar de root logger gespuugd. Je zou naar een named logger willen gaan omdat je dan later kan gaan filteren.

```suggestion
    LOG.debug(f'resolving class name from key: {name}')
```